### ISSUE TITLE
Missing Aria updates on DropDownList

### ISSUE THREAD

**Issue by CalvinConnell** [telerik/kendo-angular#1741]
## I'm submitting a...

* Bug report 

## Current behavior
Selecting different items in a DropDownList does not alert screen-reader user on status change.
Opening the droplist only notifies the user of one of the selected. (You might be using aria-labelledby where you should be using aria-label={{state}})

## Expected behavior
Use aria-live or other method to update aria labels based on selected item.

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/angular-ry4dem?file=app/app.component.ts

## Environment

Package versions:

Browser:

- Chrome (desktop) version XX

System:

**Comment by tsvetomir** [comment]
Related issue telerik/kendo-angular#1070
